source/frontend/package.json

```diff
@@ -7,6 +7,7 @@
     "@react-keycloak/web": "3.4.0",
     "@reduxjs/toolkit": "1.8.6",
     "@types/polylabel": "1.0.5",
+    "@xstate/react": "^3.2.2",
```

Comment:
@FuriousLlama our best practice for these is to lock the version to avoid inadvertent version bumps affecting application functionality.